### PR TITLE
Kornelia

### PR DESCRIPTION
**Pull request by Kornelia-K**

### CODE REVIEW

File: server/models/collection.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const mongoose = require('mongoose');
+
+const collectionSchema = new mongoose.Schema({
+    title: String,
+    list: [String],
```
Comment (BohdanOne) [inline server/models/collection.js]: name list is not very descriptive.. I think memos would work better here

File: server/models/collection.js [hunk 2]
```diff
@@ -1,16 +1,30 @@
 const mongoose = require('mongoose');
+const Joi = require('@hapi/joi');
 
 const collectionSchema = new mongoose.Schema({
+    
     title: {
-        type: String,
-        required: true
+        type: String,
```
Comment (BohdanOne) [inline server/models/collection.js]: add validation matching those from Joi

File: server/models/collection.js [hunk 3]
```diff
@@ -1,16 +1,30 @@
...
+const Joi = require('@hapi/joi');
 
 const collectionSchema = new mongoose.Schema({
+    
     title: {
-        type: String,
-        required: true
+        type: String, 
        },
     listOfMemos : {
```
Comment (BohdanOne) [inline server/models/collection.js]: still name like 'memos' would work just fine

File: server/models/collection.js (hunk 2 above)
Comment (Kornelia-K) [reply server/models/collection.js]: What does it mean?

Comment (BohdanOne) [reply server/models/collection.js]: in joi validation method you expect title to be required and between 5-255 characters, you should have the same restrictions implemented in collectionSchema

**Pull request merged**